examples/common/python/connectors/direct/tcs_listener/tcs_listener.py

```diff
@@ -104,26 +117,33 @@ def _process_request(self, input_json_str):
             response['error']['message'] = 'Error: Method has to be of type string'
             return response
 
-        if ((input_json['method'] == "WorkOrderSubmit") or
-                (input_json['method'] == "WorkOrderGetResult")):
-            return self.workorder_handler.process_work_order(input_json_str)
-        elif("WorkOrderReceipt" in input_json['method']):
-            return self.workorder_receipt_handler.workorder_receipt_handler(input_json_str)
+        if ("WorkOrderReceipt" in input_json['method']):
+            return self.workorder_receipt_handler.workorder_receipt_handler(
+                input_json_str)
         elif ("Worker" in input_json['method']):
-            return self.worker_registry_handler.worker_registry_handler(input_json_str)
+            return self.worker_registry_handler.worker_registry_handler(
+                input_json_str)
         elif ("EncryptionKey" in input_json['method']):
-            return self.worker_encryption_key_handler.process_encryption_key(input_json_str)
+            return self.worker_encryption_key_handler.process_encryption_key(
+                input_json_str)
         else:
-            response['error']['message'] = 'Error: Invalid method field'
-            return response
+            logger.info("Received Work Order request : %s",
+                        input_json['method'])
+            # save the full json for WorkOrderSubmit
+            input_json["params"]["raw"] = input_json_str
+
+            data = json.dumps(input_json).encode('utf-8')
+            response = JSONRPCResponseManager.handle(data, self.dispatcher)
```

Comment:
If a method name is unsupported will this be handled by JSONRPCResponseManager ?

Reply:
Yes, dispatcher would response with an error like

```json
{"error": {"code": -32601, "message": "Method not found"}, "id": 123, "jsonrpc": "2.0"}
```

Reply:
It would be nice to point to the documentation reference if any in the code comment for such assumptions. This will help future developers to understand the code better.